app/src/main/java/com/example/nostack/controllers/EventController.java

```diff
@@ -41,20 +43,21 @@ public EventController() {
 
     public Task<QuerySnapshot> getAllEvents() {
         return eventCollectionReference
-                .orderBy("startDate", Query.Direction.ASCENDING)
+                .orderBy("createdDate", Query.Direction.DESCENDING)
```

Comment:
wouldn't it be better for the closest event to be shown rather than creation date?

Reply:
oh right hehe i was just testing